### ISSUE TITLE
(Bug report) Docker latest tag isn't updating 

### ISSUE THREAD

**Issue by revulent** [TriliumNext/Notes#580]
### TriliumNext Version

v0.90.11-beta

### What operating system are you using?

Other (specify below)

### What is your setup?

Server access only

### Operating System Version

Docker

### Description

https://hub.docker.com/r/triliumnext/notes/tags

The latest tag wasn't updated with v0.90.11-beta

**Comment by eliandoran** [comment]
@revulent , I believe this is by design, but @perfectra1n might know more.
`latest` is same as `stable`, so it won't update unless the release does not contain the `-beta` suffix.
I think it's in line with how GitHub treats it: going to https://github.com/TriliumNext/Notes/releases/latest will go to the stable 0.90.8 instead of 0.90.11-beta.

**Comment by hulmgulm** [comment]
On the github docker repository, it points to previous beta:
![image](https://github.com/user-attachments/assets/38145721-8eae-469e-9ade-c8783331bdc7)

**Comment by perfectra1n** [comment]
Yeah this is by design. The `latest` tag will point the same as `stable`, so that for those that just do:
```
docker pull triliumnext/notes
```
will have the latest stable version of TriliumNext. The latest tag is pointing to that beta currently because I made that change after that beta had been released.

Maybe we need a `beta` tag to point to the latest beta? 😅
Or at what point do we say "just take a look at the docker tags to see what you want"?

**Comment by revulent** [comment]
Since the current latest tag points to the earlier beta, I figured it was
supposed to point to the latest beta. Since it points to a beta currently,
it probably should point to the latest beta until the next stable is
released but that is just my opinion.

**Comment by perfectra1n** [comment]
Yeah, that's a fine assumption to make, and hopefully won't be as confusing anymore once a new "stable" build of TN comes out.

> Since it points to a beta currently, it probably should point to the latest beta until the next stable is released but that is just my opinion.

I can make it do so for this 1-off time, but I think that a new stable build is pretty close so I'm not sure how long it would last for!

**Comment by eliandoran** [comment]
@perfectra1n , might be interesting to have a tag that points to latest release (beta or stable). Not sure how to call it though, since it can either be a beta or a stable depending on which one is the latest.

**Comment by perfectra1n** [comment]
Yeah - if anyone has a tag name idea for described above ^ let me know lol. Or I can just use a `beta` tag to always point to the latest beta....

**Comment by eliandoran** [comment]
Since we plan to move away from the concept of beta releases, there is no benefit to rework the CI pipeline
